staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -528,7 +528,7 @@ loop:
 
 	watchDuration := r.clock.Since(start)
 	if watchDuration < 1*time.Second && eventCount == 0 {
-		return fmt.Errorf("very short watch: %s: Unexpected watch close - watch lasted less than a second and no items received", r.name)
+		klog.V(4).Infof("very short watch: %s: Unexpected watch close - watch lasted less than a second and no items received", r.name)
```

Comment:
this is a behavior change. will the caller or the stack be OK with just logging, instead of error out ?